packages/block-library/src/categories/index.php

```diff
@@ -22,6 +22,7 @@ function render_block_core_categories( $attributes ) {
 		'orderby'      => 'name',
 		'show_count'   => ! empty( $attributes['showPostCounts'] ),
 		'title_li'     => '',
+		'hide_empty'   => ! empty( $attributes['showEmpty'] ),
```

Comment:
Thank you @mauriac  for working on this. It looks like the changes are showing inverted behaviour than expected. When I mark `Show empty categories` as checked and publish the post. It doesn't show me the empty categories. While I mark `Show empty categories` as unchecked it shows the empty categories in the published post. 

Could you please verify the behaviour?  Thank you.

Reply:
Thanks for testing and raising this one. I have fixed it. feel free to test it again.

Thanks a lot.

Reply:
Thank you @mauriac once again. Changes are working as expected. 👍🏻